frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/RecentlyUsedSegments/RecentlyUsedSegmentChip.tsx

```diff
@@ -0,0 +1,51 @@
+import type React from 'react';
+import type { ISegment } from 'interfaces/segment';
+import AddIcon from '@mui/icons-material/Add';
+import { Chip, styled, Tooltip } from '@mui/material';
+
+interface IRecentlyUsedSegmentChipProps {
+    segment: ISegment;
+    setSegments: React.Dispatch<React.SetStateAction<ISegment[]>>;
+}
+
+const StyledChip = styled(Chip)(({ theme }) => ({
+    background: 'transparent',
+    height: 'auto',
```

Comment:
Do you need this for some reason? Isn't the height kinda right? If not, you can set `size='small` on the chip. That might also do what you need if there's too much padding.